packages/twenty-server/src/modules/workflow/workflow-trigger/workflow-trigger.service.ts

```diff
@@ -21,7 +22,11 @@ export class WorkflowTriggerService {
     private readonly workflowRunnerService: WorkflowRunnerService,
   ) {}
 
-  async runWorkflow(workspaceId: string, workflowVersionId: string) {
+  async runWorkflow(
+    workspaceId: string,
+    workflowVersionId: string,
```

Comment:
see comments above, it should take the `workflowId` instead of workflowVersionId